package.json

```diff
@@ -29,12 +29,15 @@
     "prettier:text": "prettier 'README.md' 'docs/*.md' 'docs/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
     "start": "node ./bin/iotagent-json",
     "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit  --color true",
+    "test:functional": "nyc --reporter=text mocha --recursive 'test/functional/*.js' --reporter spec --timeout 5000 --ui bdd --exit  --color true",
```

Comment:
A GitAction for this test should be included.

Reply:
In that case, why don't just keep `test` target and remove `test:functional`?

Not sure if having overlapping targets is a good idea...